src/targets/github.ts

```diff
@@ -127,10 +127,21 @@ export class GitHubTarget extends BaseTarget {
       };
     }
 
-    const { data: latestRelease } = await this.github.repos.getLatestRelease({
-      owner: this.githubConfig.owner,
-      repo: this.githubConfig.repo,
-    });
+    let latestRelease: { tag_name: string } | undefined = undefined;
+    try {
+      latestRelease = (
+        await this.github.repos.getLatestRelease({
+          owner: this.githubConfig.owner,
+          repo: this.githubConfig.repo,
+        })
+      ).data;
+    } catch (error) {
+      // if the error is a 404 error, it means that no release exists yet
+      // all other errors should be rethrown
+      if (error.status !== 404) {
```

Comment:
@BYK adjusted this here to only handle 404 errors!

Reply:
We still risk getting a 404 on bad credentials that said that would be so rare that we can keep this safely (rare because these repos are meant to be public at all times)